### PR TITLE
User guide for multi-cloud archival and retrieval and File and Block service

### PR DESCRIPTION
**Pull request by kumarashit**
**What type of PR is this?**
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespace from that line:
>
/kind new feature
> /kind bug fix
> /kind cleanup
> /kind revert change
> /kind design
> /kind documentation
> /kind enhancement

**What this PR does / why we need it**:
User guide for Cloud object archival and retrieval. Also, high level doc for File and Block service of mulit-cloud

**Which issue(s) this PR fixes**:

Fixes #

**Test Report Added?**:
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespace from that line:
> /kind TESTED
> /kind NOT-TESTED

**Test Report**:

**Special notes for your reviewer**:

### CODE REVIEW

File: content/guides/user-guides/multi-cloud/object-archival-restore/_index.md [hunk 1]
```diff
@@ -0,0 +1,80 @@
+---
+title: Archival and Retrieval
+description: "User guide for Object Archival and Retrieval"
+weight: 40
```
Comment (anvithks) [inline content/guides/user-guides/multi-cloud/object-archival-restore/_index.md:4]: Set the weight for each of the sections based on the order in which they must appear.

File: content/guides/user-guides/multi-cloud/object-archival-restore/_index.md [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+disableToc: false
+tags: ["user guide", "archival and retrieval"]
```
Comment (anvithks) [inline content/guides/user-guides/multi-cloud/object-archival-restore/_index.md:6]: Add tag `multi-cloud` for all the multi-cloud related guides.

**Review by anvithks — CHANGES_REQUESTED** [review]
Overall LGTM

File: content/guides/user-guides/multi-cloud/object-archival-restore/_index.md [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+Archiving data which is important but intermittently used, is valuable to keep it safe and also save cost and resource.
+Retrieving the archived data whenever required as per need and also retrieving it when the local data is lost is an important requirement for Availability and Business continuity.
+More details about the architecture and use cases can be found at [ObjectArchivalRestore](https://github.com/sodafoundation/architecture-analysis/tree/master/arch-design/multicloud)
+
+## Getting Started
+This guide shows how to archive objects into Cloud object storage and restore them as and when required, meeting the SLAs. This will illustrate the operations through SODA Dashboard.
+
+To refer the API specs, please check [Cloud Object Archival API Specs](https://github.com/sodafoundation/multi-cloud/blob/master/openapi-spec/swagger.yaml)
+
+> \[*NOTE*\]: Currently this feature is available only for AWS cloud
```
Comment (anvithks) [inline content/guides/user-guides/multi-cloud/object-archival-restore/_index.md:21]: nit: use the [notice shortcode](https://learn.netlify.app/en/shortcodes/notice/) to highlight notes

File: content/guides/user-guides/multi-cloud/object-archival-restore/_index.md [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+
+## Installing SODA multi-cloud
+Please refer [SODA installation using Ansible](https://docs.sodafoundation.io/soda-gettingstarted/installation-using-ansible/)
+Or Check the developer guide [multi-cloud lcoal cluster installation through repo](https://docs.sodafoundation.io/guides/developer-guides/multi-cloud/)
```
Comment (anvithks) [inline content/guides/user-guides/multi-cloud/object-archival-restore/_index.md:25]: nit: lcoal -> local

File: content/guides/user-guides/multi-cloud/object-archival-restore/_index.md [hunk 5]
```diff
@@ -0,0 +1,80 @@
...
+Archived object in cloud need to be restored or brought back to the appropriate storage class before being downloaded. This depends upon the feature and SLAs from the cloud vendors
+To restore an object:
+
+* Select the object and click on **Restore** from the **Operation** column of the object
+* From the **Restore** wizard, select the number of **Days** for which the object will be available for restore or the restore link is available
+* Select the **Retrieval Tier**. Please refer the Cloud vendor docs for more details on the **Retrieval Tier** and the SLAs and costing for the same
+* Click *OK* 
+	![multi-cloud object restore](soda_object_restore.png?raw=true)
+
+> **Once the restoration of object is inititated, it may take time for the availability of the object to be downloaded. Refer cloud vendor docs for the availability corresponding to the chosen Retrieval Tier**
```
Comment (anvithks) [inline content/guides/user-guides/multi-cloud/object-archival-restore/_index.md:75]: same as before. Use the notice shortcode.

**Review by anvithks — CHANGES_REQUESTED** [review]

File: content/guides/user-guides/multi-cloud/object-archival-restore/_index.md (hunk 2 above)
Comment (anvithks) [reply content/guides/user-guides/multi-cloud/object-archival-restore/_index.md:6]: `["user guide", "archival and retrieval"]` - `["user guide", "archival and retrieval", "multi-cloud"]`

File: content/guides/user-guides/multi-cloud/object-archival-restore/_index.md (hunk 1 above)
Comment (kumarashit) [reply content/guides/user-guides/multi-cloud/object-archival-restore/_index.md]: Done

File: content/guides/user-guides/multi-cloud/object-archival-restore/_index.md (hunk 2 above)
Comment (kumarashit) [reply content/guides/user-guides/multi-cloud/object-archival-restore/_index.md]: Done

File: content/guides/user-guides/multi-cloud/object-archival-restore/_index.md (hunk 3 above)
Comment (kumarashit) [reply content/guides/user-guides/multi-cloud/object-archival-restore/_index.md]: Done

File: content/guides/user-guides/multi-cloud/object-archival-restore/_index.md (hunk 4 above)
Comment (kumarashit) [reply content/guides/user-guides/multi-cloud/object-archival-restore/_index.md]: Done

File: content/guides/user-guides/multi-cloud/object-archival-restore/_index.md (hunk 5 above)
Comment (kumarashit) [reply content/guides/user-guides/multi-cloud/object-archival-restore/_index.md]: Done

File: content/guides/user-guides/multi-cloud/object-lifecycle-management/_index.md [hunk 6]
```diff
@@ -1,7 +1,7 @@
 ---
 title: Object Lifecycle
 description: "User guide for Object Lifecycle Management."
-weight: 40
+weight: 10
 disableToc: false
 tags: ["user guide", "object lifecycle"]
```
Comment (anvithks) [inline content/guides/user-guides/multi-cloud/object-lifecycle-management/_index.md:6]: nit: add the `multi-cloud` tag

**Review by anvithks — DISMISSED** [review]
Overall LGTM.
One nit but can be merged.
Thanks Ashit.

File: content/guides/user-guides/multi-cloud/object-lifecycle-management/_index.md (hunk 6 above)
Comment (kumarashit) [reply content/guides/user-guides/multi-cloud/object-lifecycle-management/_index.md]: resolved. Please check

**Review by anvithks — APPROVED** [review]
LGTM

**Review by skdwriting — APPROVED** [review]
LGTM

**Pull request merged**